sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Microsoft.Azure.Management.Attestation.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>Attestation.Tests</PackageId>
+    <Version>1.0.0</Version>
+    <Description>Attestation.Tests Class library</Description>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.Attestation.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.ResourceManager">
+      <HintPath>..\..\..\..\..\..\Users\shlei\.nuget\packages\microsoft.azure.management.resourcemanager\1.9.1-preview\lib\net452\Microsoft.Azure.Management.ResourceManager.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
```

Comment:
@shleiAmy please delete line 15-19, the reference to microsoft.azure.management.resourcemanager should be defined in AzSdk.test.reference.props

Reply:
removed 
 <ItemGroup>
    <Reference Include="Microsoft.Azure.Management.ResourceManager">
      <HintPath>..\..\..\..\..\..\Users\shlei\.nuget\packages\microsoft.azure.management.resourcemanager\1.9.1-preview\lib\net452\Microsoft.Azure.Management.ResourceManager.dll</HintPath>
    </Reference>
  </ItemGroup>